### PR TITLE
Implemented syscall pathway for the select system call

### PR DESCRIPTION
**Pull request by jesings**

### CODE REVIEW

**Review by rennergade — COMMENTED** [review]
looks good!

**Comment by kapkic** [comment]
LGTM!

**Review by kapkic — APPROVED** [review]

File: src/trusted/service_runtime/nacl_syscall_common.c [hunk 1]
```diff
@@ -5066,3 +5066,161 @@ int32_t NaClSysFcntlSet (struct NaClAppThread *natp,
...
+    safeposixexceptfds = fd_set_fd_translator_tolind(nap, naclexceptfds, nfds, &max_fd);
+
+    if ((long) safeposixexceptfds < 0) {
+      NaClLog(2, "NaClSysBind could not translate fds in exceptfds %d\n", -NACL_ABI_EFAULT);
+      retval = (long) safeposixexceptfds;
+      goto cleanup;
+    }
+  }
+
+  if(nacltimeout) {
```
Comment (rennergade) [inline src/trusted/service_runtime/nacl_syscall_common.c:5178]: should this just be `timeout`

Comment (kapkic) [reply src/trusted/service_runtime/nacl_syscall_common.c:5178]: If we don't have any other timeouts that may cause a confusion, sure!

Comment (jesings) [reply src/trusted/service_runtime/nacl_syscall_common.c:5178]: Yes, you're right

**Pull request merged**